flight/PiOS/Common/pios_uavtalkrcvr.c

```diff
@@ -0,0 +1,171 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup  PIOS_UAVTALKRCVR Receiver-over-UAVTALK Input Functions
+ * @brief	Code to read the channels within the Receiver UAVObject
+ * @{
+ *
+ * @file       pios_uavtalkrcvr.c
+ * @author     The OpenPilot Team, http://www.openpilot.org Copyright (C) 2010.
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2013
+ * @author     dRonin, http://dronin.org Copyright (C) 2015-2018
+ * @brief      GCS/UAVTalk Input functions
+ * @see        The GNU Public License (GPL) Version 3
+ *
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, see <http://www.gnu.org/licenses/>
+ */
+
+/* Project Includes */
+#include "pios.h"
+
+#if defined(PIOS_INCLUDE_UAVTALKRCVR)
+
+#include "pios_uavtalkrcvr_priv.h"
+
+#ifndef PIOS_UAVTALKRCVR_TIMEOUT_MS
+#define PIOS_UAVTALKRCVR_TIMEOUT_MS			350
+#endif
+
+static UAVTalkReceiverData uavreceiverdata;
+
+/* Provide a RCVR driver */
+static int32_t PIOS_UAVTALKRCVR_Get(uintptr_t rcvr_id, uint8_t channel);
```

Comment:
Why the mix of caps and lower all through?

Reply:
Existing code.

Reply:
PIOS_0UAVTALKRCVR vs PIOS_uavtalkrcvr? :confused:

Reply:
Existing capitalization.  It was just a series of searches and replaces :/  (With a little bit of manual handling of a few bad things already, like some of the RCVR_RCVR's :P)